helm/charts/spar-mapper-api/values.yaml

```diff
@@ -75,14 +74,15 @@ tolerations: []
 affinity: {}
 
 envVars:
-  SPAR_G2PCONNECT_MAPPER_DB_HOSTNAME: '{{ include "postgresql.v1.primary.fullname" .Subcharts.postgresql }}'
-  SPAR_G2PCONNECT_MAPPER_DB_USERNAME: "sparuser"
-  SPAR_G2PCONNECT_MAPPER_OPENAPI_ROOT_PATH: "/mapper/v1"
+  SPAR_MAPPER_DB_HOSTNAME: "postgres-postgresql.postgres"
+  SPAR_MAPPER_DB_USERNAME: "sparuser"
+  SPAR_MAPPER_DB_DBNAME: "spardb"
+  SPAR_MAPPER_OPENAPI_ROOT_PATH: "/mapper/v1"
 
 envVarsFrom:
-  SPAR_G2PCONNECT_MAPPER_DB_PASSWORD:
+  SPAR_MAPPER_DB_PASSWORD:
     secretKeyRef:
-      name: '{{ include "postgresql.v1.primary.fullname" .Subcharts.postgresql }}'
+      name: "postgres-postgresql"
```

Comment:
These changes shouldn't be hardcoded. I think these can be reverted while keeping the ENV prefix. DB_USERNAME & DB_HOSTNAME can be entirely removed I think. If it is required to be added, it is better to do that in `scripts/values.yaml`.